pywebpush/__main__.py

```diff
@@ -0,0 +1,54 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
`__main__.py` is an official thing: it's for supporting python -m <module> where <module>'s a package

I like following the pattern of having as less code as possible in special dunder modules like `__init__.py`, restricting them to basically just imports, even in smaller projects, but that's more of a personal style thing rather than a strict standard.